runtime/android/app/src/main/cpp/sherpa-ncnn/csrc/lstm-model.cc

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
Can we reuse the code from `sherpa-ncnn/csrc/lstm-model.cc`?

Reply:
Yes, maybe we can soft link to these source files.

Reply:
Fixed